src/Sarif/Errors.cs

```diff
@@ -509,6 +510,28 @@ public static RuntimeConditions LogUnhandledEngineException(IAnalysisContext con
             return RuntimeConditions.ExceptionInEngine;
         }
 
+        public static void LogIncompatibleRule(IAnalysisContext context, string ruleId, string incompatibleRuleId)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            context.Logger.LogConfigurationNotification(
```

Comment:
>LogConfigurationNotification [](http://example.com/codeflow?start=27&length=28)

Add the updated string as a comment here. Please look at the code in this file to observe how carefully these comments are formatted (they are 'columnized').

The current configuration enables rules that are not compatible ('{0}' has declared that it is not compatible with '{1}'). You can selectively disable one of the rules using an updated XML configuration (passed by the --config argument).